client/components/backup-storage-space/storage-usage-help-popover.tsx

```diff
@@ -51,8 +51,8 @@ const StorageHelpPopover: React.FC< OwnProps > = ( {
 				<h3> { translate( 'Backup archive size' ) }</h3>
 				<p>
 					{ translate(
-						'Based on the current size of your site, Jetpack will save {{strong}}%(forecastInDays)d day of full backup{{/strong}}.',
-						'Based on the current size of your site, Jetpack will save {{strong}}%(forecastInDays)d days of full backups{{/strong}}.',
+						'Based on the current size of your site, Jetpack will save up to {{strong}}%(forecastInDays)d day of full backup{{/strong}}.',
```

Comment:
This reads kind of strange, I wonder if we should reword the singular case to
`1 full backup` instead of `1 day of full backup`

Reply:
Makes sense. Thanks for the suggestion.
I just updated it :)